vigor/views.py

```diff
@@ -46,4 +61,4 @@ def add_item(request):
     return HttpResponse("Should include a form with a redirect to dash on submit. The data entered should also appear on the dash")
 
 def users(request):
-    return HttpResponse(Users.objects.all())
+    return HttpResponse(User.objects.all())
```

Comment:
purely for testing purposes until I create an admin and put the Users in a view on the admin page. That is going to be coming up later.